src/components/FileInput.js

```diff
@@ -176,7 +181,14 @@ class FileInput extends Component {
           }}
           onChange={this.selectFile}
         />
-        {this.props.displayMetadata && renderMetadata}
+        <button onClick={this.openFileDialog}>Select File</button>
+        <div className={!isDragging && 'brainhub-file-input__dropInfo--hidden' || ''}>
```

Comment:
This drop and the button has to be removed (it was substituted with the drop area).